clients/python/src/kaskada/api/session.py

```diff
@@ -201,6 +202,56 @@ def manager_grpc_port(self, port: int):
         self.manager_configs["-grpc-port"] = port
         return self
 
+    def object_store_type(self, object_store_type: str):
```

Comment:
Thought: Rather than trying to create the public API for the object store (which probably merits some discussion in terms of how the user would expect to configure), it may be easier to modify the `manager_configs` directly. 

Couldn't we just do:

```
builder = LocalBuilder()
builder.manager_configs["-object-store-type"] = "local"
...
session = builder.build()
```

We could make a method that makes it easier to add manager flags -- `builder.with_manager_configs([("-object-store-type", "s3"), ...])`.

The main benefit is that it makes it clearer that these options and configuration is still in flux. Creating the `ObjectStoreConfig` type and using it makes it seem more stable / long-term than it likely is.

Reply:
This sounds like a good idea. Let me verify that it works 👍🏻

Reply:
This works as a simple change, and the usage would look like:
```python
import kaskada

from kaskada.api.session import LocalBuilder
session = LocalBuilder().with_manager_configs([\
    ("-object-store-type", "local"),\
    ("-object-store-path", "/Users/kevin.nguyen/Github/kaskada/examples3")\
 ]).build()
```

Reply:
I like that. Maybe add call it `with_manager_args` or `with_manager_flags`? Then add some python doc indicating that this adds arguments to be passed directly to the manager executable.